desc/grid.py

```diff
@@ -1393,11 +1468,19 @@ def _create_nodes(self, L, M, N, NFP=1, axis=False, node_pattern="jacobi"):
             node spacing, based on local volume around the node
 
         """
+        self._L = L
+        self._M = M
+        self._N = N
+        self._NFP = NFP
```

Comment:
These look like artifacts from merge conflict, should remove them.

Reply:
Resolved.